Documentation/HowToUseETW.md

```diff
@@ -0,0 +1,46 @@
+# Supported
+ETW logs are available on CoreCLR only currently.
+
+
+# How to Collect ETW logs use logman
+
+
+## Start the logging
+
+ * Option 1: Generate a single file that does circular logging with Verbose level, For example
+ 
+ ```
+logman start WCFETWTracing -p "Microsoft-Windows-Application Server-Applications" 0xFFFFFFFF  0x5 -bs 64 -nb 120 320 -ets -ct perf -f bincirc -max 500 -o c:\temp\WCFEtwTrace.etl
```

Comment:
I recommend the document be structured so it is clear how to use ETW as an end-user to debug an app, and how to use ETW as an OSS developer to debug product issues.  In both cases, we need to be clear which commands are executed on the client machine and which are executed on a server machine.  I think a typical debug scenario will want traces from both client and server, for both end users and OSS users.